iopipe/report.py

```diff
@@ -37,6 +37,7 @@ def __init__(self, config, context):
         self.config = config
         self.context = context
         self.custom_metrics = []
+        self.labels = []
```

Comment:
Shouldn't the `[]` be a `set()`?

Reply:
Yes! Will do and add tests appropriately